content/maxwell3_fdem/wave_number.rst

```diff
@@ -0,0 +1,84 @@
+.. _waveNumber:
+
+Wave Number
+==========
```

Comment:
Annoying detail: this title underline is too short, causing the build to be unstable